mergify_engine/tests/unit/test_synchronise.py

```diff
@@ -0,0 +1,70 @@
+# -*- encoding: utf-8 -*-
```

Comment:
this uses the American version of `synchronize` with a z, you should rename the file :)